package.json

```diff
@@ -7,7 +7,7 @@
     "instrumentationKey": "__AIKEY__",
     "icon": "assets/icon.png",
     "engines": {
-        "vscode": "^1.34.0"
+        "vscode": "^1.43.0"
```

Comment:
Would it be worthwhile to add this into the readme ( if it's not already added)

Reply:
sure, can do

Reply:
Added it as a badge